firstvoices/backend/views/search/base_search_views.py

```diff
@@ -256,6 +256,78 @@
                     ),
                 ],
             ),
+            OpenApiParameter(
+                name="hasAudio",
+                description="Filter documents that have related audio.",
+                required=False,
+                default=False,
+                type=bool,
+                examples=[
+                    OpenApiExample(
+                        "True",
+                        value=True,
+                        description="Searches for documents that have related audio.",
+                    ),
+                    OpenApiExample(
+                        "False",
+                        value=False,
+                        description="No hasAudio filter applied.",
+                    ),
+                    OpenApiExample(
+                        "Oranges",
+                        value=False,
+                        description="Invalid input, defaults to false.",
+                    ),
+                ],
+            ),
+            OpenApiParameter(
+                name="hasImage",
+                description="Filter documents that have related images.",
+                required=False,
+                default=False,
+                type=bool,
+                examples=[
+                    OpenApiExample(
+                        "True",
+                        value=True,
+                        description="Searches for documents that have related images.",
+                    ),
+                    OpenApiExample(
+                        "False",
+                        value=False,
+                        description="No hasImage filter applied.",
+                    ),
+                    OpenApiExample(
+                        "Oranges",
+                        value=False,
+                        description="Invalid input, defaults to false.",
+                    ),
+                ],
+            ),
+            OpenApiParameter(
+                name="hasVideo",
+                description="Filter documents that have related videos.",
+                required=False,
+                default=False,
+                type=bool,
+                examples=[
+                    OpenApiExample(
+                        "True",
+                        value=True,
+                        description="Searches for documents that have related videos.",
+                    ),
+                    OpenApiExample(
+                        "False",
+                        value=False,
+                        description="No hasVideo filter applied.",
+                    ),
+                    OpenApiExample(
+                        "Oranges",
+                        value=False,
+                        description="Invalid input, defaults to false.",
+                    ),
```

Comment:
Did we already switch all the boolean filters to allow true/false/unset? If so I think these docs need to make that clear-- I would expect the default and the "Oranges" value to be null, not false

Reply:
We haven't added that functionality yet. Currently we only allow true or false. The ticket to add unset is [FW-5183](https://firstvoices.atlassian.net/browse/FW-5183).